cli/testdata/server-config.yaml.golden

```diff
@@ -532,13 +577,13 @@ notifications:
     # (default: <unset>, type: string)
     from: ""
     # The intermediary SMTP host through which emails are sent.
-    # (default: localhost:587, type: host:port)
+    # (default: <unset>, type: host:port)
```

Comment:
Is unsetting the default for this deprecated `smarthost` option backward compatible? I understand that it now has a UseInstead attribute set to a new option with a default. Will the UseInstead attribute mean that this option inherits the default of its replacement?

I ask because the issue that you reference in the description of this PR mentions that we'd like to stay BC for now.

Reply:
Actually. I get it. As long as we set the default in one place it references the same option in memory.

Reply:
Yep! Beat me to it 😄